packages/mobile/src/screens/chat-screen/ChatListItem.tsx

```diff
@@ -105,34 +52,65 @@ export const ChatListItem = ({ chatId }: { chatId: string }) => {
   return (
     <TouchableHighlight onPress={handlePress}>
       {otherUser ? (
-        <View style={styles.root}>
-          <View style={styles.contentRoot}>
-            <View style={styles.userContainer}>
+        <Flex
+          column
+          pv='l'
+          ph='xl'
+          backgroundColor='white'
+          borderBottom='default'
+          w='100%'
+        >
+          <Flex row justifyContent='space-between' w='100%' gap='s'>
+            <Flex row flex={1} justifyContent='space-between'>
```

Comment:
this `flex={1}` still weirds me out... `w=100%` no worko? feels like this should be a default?
watevs, let's chat tmr in the client sync maybe.

Reply:
it's short for flexGrow: 1, flexShrink: 1, flexBasis: 0 which makes it take up available space in the flex parent